src/components/SystemManagedList.js

```diff
@@ -333,11 +336,14 @@ export default class SystemManagedList extends Component {
       ? components.list.filter(item => isEmpty(item.changes))
       : components.list.map(({ changes, ...keepAttrs }) => keepAttrs)
     if (this.hasChanges()) {
-      dispatch(
-        uiBrowseUpdateList({
-          updateAll: refreshedData
-        })
-      )
+      switch (this.storeList) {
```

Comment:
feels like this should be delegated to the subclass. Add a new function `updateList` with a default empty implementation the override in the subclasses (is there subclass?). or plug in a loader function as a prop?